diagnostic.md

```diff
@@ -65,5 +76,6 @@ item elements) to the body DOM node, then hides it.
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
+$("div").on('click')
```

Comment:
Correct. You will have something happen though, typically a function after 'click'.